src/actions/public/simulateDepositETH.test.ts

```diff
@@ -0,0 +1,18 @@
+import { expect, test } from 'vitest'
+import { publicClientMainnet } from '../../_test/utils'
+import { base } from '@roninjin10/rollup-chains'
+import { simulateDepositETH } from './simulateDepositETH'
+
+test('default', async () => {
+  expect(
+    await simulateDepositETH(publicClientMainnet, {
```

Comment:
@zencephalon do we want to test that the return value here can be passed to writeDepositETH?

Reply:
yeah, I'm attempting to in a stashed commit